Upstream: Unknown, should be
Reason: Fixes compilation under gcc-10 which has -fno-common by default

diff --git a/Config/exports.h b/Config/exports.h
index 2c5edec..1dafc36 100644
--- a/Config/exports.h
+++ b/Config/exports.h
@@ -22,7 +22,7 @@
 extern exports	exports_nfslist;
 /* Options cache */
 extern int	exports_opts;
-const char      *export_path; 
+extern const char      *export_path; 
 extern uint32 	export_fsid;
 extern uint32   export_password_hash;
 
diff --git a/Config/exports.y b/Config/exports.y
index f30e8af..6b8fbf9 100644
--- a/Config/exports.y
+++ b/Config/exports.y
@@ -694,7 +694,7 @@ static e_host* find_host(struct in_addr remote, e_item *item,
 
 /* options cache */
 int exports_opts = -1;
-const char *export_path = NULL; 
+extern const char *export_path = NULL; 
 uint32 export_fsid = 0;
 uint32 export_password_hash = 0;
 
diff --git a/Config/y.tab.c b/Config/y.tab.c
index b0761f4..3a61594 100644
--- a/Config/y.tab.c
+++ b/Config/y.tab.c
@@ -2248,7 +2248,7 @@ static e_host* find_host(struct in_addr remote, e_item *item,
 
 /* options cache */
 int exports_opts = -1;
-const char *export_path = NULL; 
+extern const char *export_path = NULL; 
 uint32 export_fsid = 0;
 uint32 export_password_hash = 0;
 
